packages/next/src/index.ts

```diff
@@ -2606,18 +2620,39 @@ async function getServerlessPages(params: {
   pagesDir: string;
   entryPath: string;
   outputDirectory: string;
+  appPathRoutesManifest?: Record<string, string>;
 }) {
-  const [pages, middlewareManifest] = await Promise.all([
+  const [pages, appPaths, middlewareManifest] = await Promise.all([
     glob('**/!(_middleware).js', params.pagesDir),
+    params.appPathRoutesManifest
+      ? glob('**/page.js', path.join(params.pagesDir, '../app'))
+      : Promise.resolve({}),
     getMiddlewareManifest(params.entryPath, params.outputDirectory),
   ]);
 
+  const normalizedAppPaths: typeof appPaths = {};
+
+  if (params.appPathRoutesManifest) {
+    for (const [entry, normalizedEntry] of Object.entries(
+      params.appPathRoutesManifest || {}
```

Comment:
You can remove the `if` on 2635 or remove the `|| {}` on line 2637

Reply:
Updated in https://github.com/vercel/vercel/pull/8098/commits/5a04b53b8eb34e8ea4e07cfb63ff307c81b9df69